jest-preset.js

```diff
@@ -1,9 +1,17 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+const esModules = [
+  '@cloudscape-design',
+
+  // `d3-color` and `d3-scale` packages (plus their transitive dependencies) are using the ESM format.
+  // We need to transform them to have them working in tests.
+  'd3-.*',
+  'internmap',
+].join('|');
 module.exports = {
   transform: {
-    'node_modules/@cloudscape-design/.+\\.js$': require.resolve('./js-transformer'),
+    [`node_modules/(${esModules})/.+\\.js$`]: require.resolve('./js-transformer'),
```

Comment:
Should this be an array? The [jest documentation](https://jestjs.io/docs/configuration#transform-objectstring-pathtotransformer--pathtotransformer-object) has the definition as `object<string, pathToTransformer | [pathToTransformer, object]>`

Reply:
It's not an array. The brackets are used to set the computed property name ([reference on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#computed_property_names)).

It's get's compiled to:
```
transform: {
   "node_modules/(@cloudscape-design|d3-.*|internmap)/.+\\.js$": require.resolve('./js-transformer')
}
```

Reply:
All good, post holiday brain on my side 😅🚀